examples/gno.land/r/nt/boards2/v1/public.gno

```diff
@@ -1,10 +1,22 @@
 package boards2
 
 import (
+	"regexp"
 	"std"
+	"strconv"
 	"strings"
 )
 
+const (
+	// MaxBoardNameLength defines the maximum length allowed for board names.
+	MaxBoardNameLength = 50
+
+	// MaxThreadTitleLength defines the maximum length allowed for thread titles.
+	MaxThreadTitleLength = 100
+)
+
+var reBoardName = regexp.MustCompile(`(?i)^[a-z]+[a-z0-9_\-]{2,50}$`)
```

Comment:
Added this regexp to validate board names. The idea is that names would look similar to Reddit post names where a restricted set of characters are allowed.

Names would look like:
- xyz
- TheNumber42
- theNumber42
- the-number-42
- the_number_42